enhancements/monitoring/user-workload-monitoring.md

```diff
@@ -218,15 +238,13 @@ For this use case just one system is critical for alerts to work:
 ##### Developer perspective
 
 Alerts/Recording Rules Aggregation implies that the user potentially chooses whether recording or alerting rules are being deployed on the user workload Prometheus or Thanos Ruler.
-In order to retrieve the full list of user workload recording alerting and recording rules a fan-out request has to be executed against user workload Prometheus and Thanos Ruler
-and the response has to be merged for final display. The fan-out/merge proxy logic is suggested to be implemented in the console backend (being a Go based proxy server).
+In order to retrieve the full list of user workload recording alerting and recording rules the existing Thanos Querier gets support for the Prometheus `/api/v1/rules` and `/api/v1/alerts` API. Internally it fans out to the existing in-cluster Prometheus, the user workload monitoring Prometheus, and ThanosRuler. See [7] and [8] for further implementation details.
```

Comment:
I thought we wanted only rules API :thinking:

Reply:
Ah indeed, yes, I checked this in console code that we don't use `/api/v1/alerts`, but let's do a final verification with @kyoto.

Reply:
removed `/api/v1/alerts`

Reply:
👍 Console doesn't use `/api/v1/alerts`